lib/sisimai/bite/email/facebook.rb

```diff
@@ -0,0 +1,274 @@
+module Sisimai::Bite::Email
+  module Facebook
+    # Sisimai::Bite::Email::Facebook parses a bounce email which created by Facebook.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Facebook.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :from    => %r/\AFacebook [<]mailer-daemon[@]mx[.]facebook[.]com[>]\z/,
+        :subject => %r/\ASorry, your message could not be delivered\z/,
+      }
+      Re1 = {
+        :begin   => %r/\AThis message was created automatically by Facebook[.]\z/,
+        :rfc822  => %r/\AContent-Disposition: inline\z/,
+        :endof   => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+
+      # http://postmaster.facebook.com/response_codes
+      # NOT TESTD EXCEPT RCP-P2
+      ReFailure = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->